package.json

```diff
@@ -8,23 +8,59 @@
   },
   "dependencies": {
     "antd": "^2.13.4",
+    "autoprefixer": "7.1.2",
+    "babel-core": "6.25.0",
+    "babel-eslint": "7.2.3",
+    "babel-jest": "20.0.3",
+    "babel-loader": "7.1.1",
+    "babel-preset-react-app": "^3.0.3",
+    "babel-runtime": "6.26.0",
+    "case-sensitive-paths-webpack-plugin": "2.1.1",
+    "chalk": "1.1.3",
+    "css-loader": "0.28.4",
+    "dotenv": "4.0.0",
+    "eslint": "4.4.1",
+    "eslint-config-react-app": "^2.0.1",
+    "eslint-loader": "1.9.0",
+    "eslint-plugin-flowtype": "2.35.0",
+    "eslint-plugin-import": "2.7.0",
+    "eslint-plugin-jsx-a11y": "5.1.1",
+    "eslint-plugin-react": "7.1.0",
+    "extract-text-webpack-plugin": "3.0.0",
+    "file-loader": "0.11.2",
+    "fs-extra": "3.0.1",
+    "html-webpack-plugin": "2.29.0",
+    "jest": "20.0.4",
+    "object-assign": "4.1.1",
+    "postcss-flexbugs-fixes": "3.2.0",
+    "postcss-loader": "2.0.6",
+    "promise": "8.0.1",
     "react": "^16.0.0",
+    "react-dev-utils": "^4.1.0",
     "react-dom": "^16.0.0",
-    "react-scripts": "1.0.14"
+    "react-redux": "^5.0.6",
+    "redux": "^3.7.2",
+    "redux-actions": "^2.2.1",
+    "style-loader": "0.18.2",
+    "sw-precache-webpack-plugin": "0.11.4",
+    "url-loader": "0.5.9",
+    "webpack": "3.5.1",
+    "webpack-dev-server": "2.8.2",
+    "webpack-manifest-plugin": "1.2.1",
+    "whatwg-fetch": "2.0.3"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "start": "node scripts/start.js",
+    "build": "node scripts/build.js",
+    "test": "node scripts/test.js --env=jsdom",
     "release": "standard-version",
-    "eject": "react-scripts eject",
     "eslint": "eslint .",
     "commitmsg": "commitlint -e",
     "precommit": "lint-staged"
   },
   "lint-staged": {
     "*.js": [
-      "eslint --cache --max-warnings 0"
```

Comment:
What was the reason to remove that?

Reply:
I was having some weird warn. I don't think warns should avoid a feature to be implemented otherwise should be mapped as errors.

Reply:
Weird is a generic argument. If you're having warnings you should fix or discuss if the rules are necessary.

Reply:
the weird word was not even my point... Hehe but I will take a look as soon as I have som time :D

Reply:
Nice @wellyal thanks for your patience! hahahaha

Reply:
done